fallback-dependencies.js

```diff
@@ -4,6 +4,10 @@ const { spawnSync, spawn } = require('child_process')
 let pkgPath = process.argv[1] // full path of postinstall script being executed, presumably buried in node_modules in your app
 pkgPath = pkgPath.split('node_modules')[0] // take only the part preceding node_modules
 const pkg = require(pkgPath + 'package.json') // require the package.json in that folder
```

Comment:
also bump package.json version to 1.1.0 and regenerate the package-lock after doing so